e2e/e2e_test.go

```diff
@@ -413,10 +414,10 @@ spec:
 			if !assert.NoError(t, err) {
 				t.FailNow()
 			}
-			old := ext.GetOpenAPIFile
-			defer func() { ext.GetOpenAPIFile = old }()
-			ext.GetOpenAPIFile = func(args []string) (s string, err error) {
-				return dir + "/Kptfile", nil
+			old := ext.KRMFileName
```

Comment:
Do we need to set this back to old name ? I see this pattern in some of the tests but don't see a reason why do we want to set it back.

Reply:
`KRMFileName` is a package-level variable, so changing the value in one test can lead to unexpected results in other tests. So I think it is a good practice to try to clean up the environment in each test.